sql/parser/builtins.go

```diff
@@ -382,8 +382,8 @@ var builtins = map[string][]builtin{
 			types:      typeList{timestampType},
 			returnType: DummyInterval,
 			impure:     true,
```

Comment:
I think this function and the other time functions (except `clock_timestamp`) are now pure.